src/Composer/Package/Loader/RootPackageLoader.php

```diff
@@ -93,6 +99,14 @@ public function load(array $config, string $class = 'Composer\Package\RootPackag
             }
 
             if (!isset($config['version'])) {
+                if ($this->io !== null && $config['name'] !== '__root__') {
+                    $this->io->warning(
+                        sprintf(
+                            "Composer couldn't find the root package version for %s, defaulting to `1.0.0`. See https://getcomposer.org/root-version",
```

Comment:
Sorry you were too quick I was editing :D 

```suggestion
                            "Composer could not detect the root package (%s) version, defaulting to "1.0.0". See https://getcomposer.org/root-version",
```

Reply:
Going to fix the typo